src/LinearMaps.jl

```diff
@@ -10,10 +10,6 @@ abstract type LinearMap{T} end
 Base.eltype(::LinearMap{T}) where {T} = T
 
 Base.isreal(A::LinearMap) = eltype(A) <: Real
-LinearAlgebra.issymmetric(::LinearMap) = false # default assumptions
-LinearAlgebra.ishermitian(A::LinearMap{<:Real}) = issymmetric(A)
-LinearAlgebra.ishermitian(::LinearMap) = false # default assumptions
-LinearAlgebra.isposdef(::LinearMap) = false # default assumptions
 
```

Comment:
I think these fallbacks were for people that write their own `LinearMap` subtype, and don't define these properties. A test case could include a `SimpleFunctionMap` type that does not declare these properties.

Reply:
I see, I didn't think that far. :) I'll add them back again... and add some tests. 😆